pkg/kudoctl/util/github/client.go

```diff
@@ -84,11 +85,11 @@ func NewGithubClient(cred string) (*GithubClient, error) {
 		tp.OTP = cred
 	}
 
-	return &GithubClient{client: client}, nil
+	return &Client{client: client}, nil
 }
 
 // GetMostRecentFrameworkContentDir returns the content of the most recent Framework
-func (g *GithubClient) GetMostRecentFrameworkContentDir(framework string) (*github.RepositoryContent, error) {
+func (g *Client) GetMostRecentFrameworkContentDir(framework string) (*github.RepositoryContent, error) {
```

Comment:
It's common in Go when making methods for the variable name to match the first letter of the struct the method is implemented on, so:

`func (c *Client)`. Can we rename for consistency?

Reply:
Otherwise, agree with this change (and the Kudo client change, the package name is self-descriptive)